readme.md

```diff
@@ -123,6 +180,7 @@ Your Answer:
 
 Describe some of the common git workflows for teams (fork and pull request, etc).
 
-```text
+```
+Fork, clone, git add ., git commit, git push origin 'branchname', submit pull request, merge, git pull upstream master.
```

Comment:
This is one of the common ones but I'd like to see a bit more. Check out the solution branch for additional options.